dist/js/materialize.js

```diff
@@ -220,8 +217,15 @@ jQuery.extend( jQuery.easing,
-    if (typeof define === 'function' && define.amd) {
-        define(['jquery', 'hammerjs'], factory);
+    if (typeof define === 'function' && define.amd == false) {
+        // hammerjs will already be publicly available
+        define(['jquery'], function($, Hammer) {
```

Comment:
so, it's been a while since the last time i worked with this repo so i investigated a bit and just remembered what i did exactly, turns out this file is kind of outdated, because this file is compiled with frunt but i only ran the dist js compile once on the first stages of the solution, this section where talking about is the jquery.hammer.js so you can see what i really did and what worked and what i tested , i was compiling the temp grunt task rather than dest so the most updated version is in the bin directory, that file is not readable so you can take a look at the real code in js/jquery.hammer.js there you can see what i did exactly, if you have any problem just LMK ok ?